src/utils/timing.js

```diff
@@ -16,15 +15,6 @@ const start = (key: string) => {
   timingMap[key] = now();
 };
 
-const startGroup = (key: string) => {
-  timingMap[key] = now();
```

Comment:
That might explain some perf underestimations on the HTML parsing and rendering in the native HTML. And will explain the higher than expected perf boost when switching to WebView ;)